### PR TITLE
[#11] Implement logging architecture

### PR DESCRIPTION
**Pull request by semyonov-m-y**

### CODE REVIEW

**Review by holkwen — DISMISSED** [review]

**Review by zozich — CHANGES_REQUESTED** [review]
WTF? Why isn't .idea folder in .gitignore?

File: Startup_Platform.iml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Sheptytskyid) [inline Startup_Platform.iml]: please remove this file

File: src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Sheptytskyid) [inline src/main/resources/logback.xml]: please add another logback.xml for tests in src/test/resources

**Review by zozich — APPROVED** [review]
OK

**Pull request merged**